pom.xml

```diff
@@ -27,7 +27,7 @@
   <packaging>jar</packaging>
 
   <properties>
-    <beam.version>2.9.0</beam.version>
+    <beam.version>2.11.0</beam.version>
     <samza.version>0.14.1</samza.version>
```

Comment:
Are we not updating the samza version to 1.0?

Reply:
Synced up with @xinyuiscool offline, this currently is not yet updated for 1.0 version of Samza